apps/account-api/src/controllers/prometheus.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Get, HttpCode, HttpStatus, Header } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { register, collectDefaultMetrics } from 'prom-client';
+
+collectDefaultMetrics();
+
+@Controller()
+@ApiTags('metrics')
+export class PrometheusController {
```

Comment:
The metrics endpoint.

Reply:
Wish it returned JSON! Looks like there might be a method to do that if we want `register.getMetricsAsJSON()`